kotsadm/web/src/components/apps/AppVersionHistoryRow.jsx

```diff
@@ -75,12 +84,22 @@ function renderVersionAction(version, nothingToCommitDiff, app, history, deployV
 
   const isSecondaryBtn = isPastVersion || needsConfiguration || isRedeploy && !isRollback;
   const isPrimaryButton = !isSecondaryBtn && !isRedeploy && !isRollback;
+  let tooltipTip;
+  if (checkIsCurrentOrPastVersion(currentDownstreamVersion, version)) {
```

Comment:
This should allow editing the latest version, not the one currently deployed.  This is current behavior.